lib/extensions/ar_virtual.rb

```diff
@@ -552,14 +552,14 @@ def virtual_field?(name)
     def remove_virtual_fields(associations)
       case associations
       when String, Symbol
-        virtual_field?(associations) ? nil : associations
+        virtual_field?(associations) ? {} : associations)
```

Comment:
Syntax error:  extra paren.  #justPassingThrough

Reply:
thats what I get for running just local tests on one file :(